tests/conftest.py

```diff
@@ -1,2 +1,38 @@
+import sys
+from pathlib import Path
+from typing import Optional
+
+from _pytest.config import Config
+from py import path as pypath
+
 # See https://github.com/HypothesisWorks/hypothesis/pull/2567
 pytest_plugins = ['hypothesis.extra.pytestplugin']
+
+python_version = (sys.version_info.major, sys.version_info.minor)
+paths_to_ignore = {  # noqa: WPS407
```

Comment:
`types.MutableMapping`?

Reply:
It's `Final` too, isn't it?

Reply:
Oh, it's `types` no `typing`!

Maybe [types.MappingProxyType](https://docs.python.org/3/library/types.html#types.MappingProxyType) should be better!